src/scaffolding/wizard/netCore/NetCoreGatherInformationStep.ts

```diff
@@ -53,16 +53,14 @@ export class NetCoreGatherInformationStep extends GatherInformationStep<NetCoreS
             wizardContext.netCoreRuntimeBaseImage = wizardContext.platform === '.NET: ASP.NET Core' ? `${aspNetBaseImage}:${netCoreVersion.major}.${netCoreVersion.minor}` : `${consoleNetBaseImage}:${netCoreVersion.major}.${netCoreVersion.minor}`;
             wizardContext.netCoreSdkBaseImage = `${netSdkImage}:${netCoreVersion.major}.${netCoreVersion.minor}`;
 
-            if (netCoreVersion.major === 8) {
-                if (wizardContext.netCorePlatformOS === 'Windows') {
-                    // append '-preview-nanoserver-ltsc2022' for windows base images for .NET 8's new naming convention
-                    wizardContext.netCoreRuntimeBaseImage = `${wizardContext.netCoreRuntimeBaseImage}-preview-nanoserver-1809`;
-                    wizardContext.netCoreSdkBaseImage = `${wizardContext.netCoreSdkBaseImage}-preview-nanoserver-1809`;
-                } else {
-                    // append '-preview' for everything else
-                    wizardContext.netCoreRuntimeBaseImage = `${wizardContext.netCoreRuntimeBaseImage}-preview`;
-                    wizardContext.netCoreSdkBaseImage = `${wizardContext.netCoreSdkBaseImage}-preview`;
-                }
+            if (netCoreVersion.major === 9) {

```

Comment:
Only thing I'd do differently is maybe make this a constant at the top of the file, like `const NetCorePreviewVersion = 9`

Reply:
good point!